polynote-server/src/main/scala/polynote/server/package.scala

```diff
@@ -71,7 +71,9 @@ package object server {
         case _ => ZIO.none
       }.catchAll {
         err => ZStream.fromEffect(Logging.error(err)).drain
-      }.unNone.ensuring {
+      }.collect {
+        case Some(frame) => frame
```

Comment:
Is this better than `unNone`?

Reply:
No, but `unNone` seems to be gone, so...

Reply:
😮

Reply:
🤨